pandasql/core.py

```diff
@@ -175,6 +176,12 @@ def all(self):
     def __getattr__(self, attr):
         if attr in self.columns:
```

Comment:
Just a thing to remember - we will have weird/unexpected bugs if users name columns with function names

Reply:
Yes, good point. Pandas has the same problem. In such a case in Pandas, the method takes precedence over the column. I'll change this to reflect that.